### ISSUE TITLE
SMD version?

### ISSUE THREAD

**Issue by meup** [lfantoniosi/tn9k_f18a#2]
I have an MSX computer that I don't think has a lot of vertical space above the 9918A (motherboard is below keyboard).
Would you be able to design an SMD version (for the 245 IC's & decoupling caps) that requires less height?
Maybe also a FPC connector for the VGA port might help, and I believe there are SMD DIP switches as well.
I notice that the 74LVC245 comes in many possible packages; the "SO" type seems a reasonable size for hand soldering.

**Comment by hanshuebner** [comment]
I have done an SMD version in [my fork](https://github.com/hanshuebner/tn9k_f18a/tree/compact) - It looks about like this:

<img width="269" alt="image" src="https://github.com/lfantoniosi/tn9k_f18a/assets/108751/41a0d6e1-50d3-49ee-a134-4e35a23e7cc6">

<img width="248" alt="image" src="https://github.com/lfantoniosi/tn9k_f18a/assets/108751/28bf724b-f261-41cb-9601-a28277708dbe">

It does not currently have configuration switches as I believe that solder jumpers will be adequate for most use cases, but maybe I'll have some on the next revision.  Likewise for the FPC connector - I've gone for the snap-off part for now, but if there is enough demand, replacing the IDC footprint by an FPC footprint won't be hard.

The DIP-40 pins are shifted slightly to one side to accommodate the space available in the TI-99/4A.  I've got a couple of boards on order and will open a PR once I have tested them and verified that they work.
